docs(input-group): improve input group & reactive forms documentation (#1928)
IgniteUI/igniteui-docfx: en/components/input_group_reactive_forms.md

## Reactive Forms Integration
<p class="highlight">

> The Ignite UI for Angular controls can be easily integrated into Reactive Forms that provide a model-driven approach for handling form inputs.

Reviewer: ```suggestion
The Ignite UI for Angular controls can easily be used in Reactive Forms that provide a model-driven approach for handling form inputs.
```

</p>

### Demo

> The following sample demonstrates the [`igx-input-group`]({environment:angularApiUrl}/classes/igxinputgroupcomponent.html), [`igx-select`]({environment:angularApiUrl}/classes/igxselectcomponent.html) and [`igx-combo`]({environment:angularApiUrl}/classes/igxcombocomponent.html) controls being part of the Reactive Forms.

Reviewer: ```suggestion
The following sample demonstrates the [`igx-input-group`]({environment:angularApiUrl}/classes/igxinputgroupcomponent.html), [`igx-select`]({environment:angularApiUrl}/classes/igxselectcomponent.html) and [`igx-combo`]({environment:angularApiUrl}/classes/igxcombocomponent.html) controls being part of the Reactive Forms.
```
```suggestion
The following sample demonstrates the [`igx-input-group`]({environment:angularApiUrl}/classes/igxinputgroupcomponent.html), [`igx-select`]({environment:angularApiUrl}/classes/igxselectcomponent.html) and [`igx-combo`]({environment:angularApiUrl}/classes/igxcombocomponent.html) controls when used in Reactive Forms.
```

### Usage

#### First Steps

> To get started with the Reactive Forms, first you have to import all needed modules for Ignite UI components as well as the `ReactiveFormsModule`:

Reviewer: ```suggestion
Before you get started with Reactive Forms, you need to import all required modules for the Ignite UI components as well as the `ReactiveFormsModule`:
```

export class AppModule {}
```

> For a complete introduction to the Reactive Forms, follow the [Angular tutorial](https://angular.io/guide/reactive-forms).

Reviewer: ```suggestion
For a complete introduction to Reactive Forms, follow the [Angular tutorial](https://angular.io/guide/reactive-forms).
```

#### Create Form

> Use the `formGroup` property of the HTML `form` to set a model of the form, and then apply a `formControlName` to each of the input fields, which will correspond to individual model's properties.

Reviewer: ```suggestion
Use the `formGroup` property of the `form` element to set the model for the form. Next, add `formControlName` to each of the input fields, which will correspond to each individual model property.
```

    </igx-combo>
    ...
```

> The object, set to the `formGroup` property, is the form model which needs to be of type `FormGroup`. Then, following Angular tutorial for Reactive Forms, we need a `FormBuilder` in our constructor in order to define and configure different form's fields:

Reviewer: ```suggestion
The object, assigned to the `formGroup` property, is the form model which needs to be of type `FormGroup`. Then, following the Angular tutorial for Reactive Forms, we need a `FormBuilder` in our constructor in order to define and configure the various form fields:
```

}
```

> In that case the movie, the full name, the email and the genres form's fields are required and the form won't be submitted, unless they have values.

Reviewer: ```suggestion
In our case the `movie`, `fullName`, `email`, and `genres` form fields are required and the form won't be submitted, unless their values are set.
```
